mesoSPIM/src/mesoSPIM_ImageWriter.py

```diff
@@ -55,6 +56,70 @@ def check_versions(self):
 
         tifffile.TiffWriter.write = self.tiff_write # rename the entire class method if necessary
 
+    def acquisition_shape(self):
+        '''Return a tuple of acquisition shape (z,y,x)'''
+        return (self.max_frame, self.y_pixels, self.x_pixels)
+
+    def allocate_np(self):
+        '''Return a numpy array filled with zeros the shape of the acquisition array'''
+        return np.zeros(self.acquisition_shape(), dtype='uint16')
+
+    @staticmethod
+    def gb_size_of_array_shape(shape):
```

Comment:
Not critical, but maybe worth moving this function to [/src/utils/utility_functions.py](https://github.com/mesoSPIM/mesoSPIM-control/blob/development/mesoSPIM/src/utils/utility_functions.py), which is a collection of similar general-purpose functions.